.github/workflows/create-release-pr.yaml

```
@@ -35,8 +60,8 @@ jobs:
         id: release-pr
         uses: changesets/action@aba318e9165b45b7948c60273e0b72fce0a64eb9 # v1.4.7
         with:
-          branch: 'main'
-          title: 'chore(release): bump version and update changelog'
-          commit: 'chore(release): bump version and update changelog'
+          branch: ${{ steps.branch-check.outputs.branch_name }} # This branch context is ignored in this step: `https://github.com/changesets/action/blob/50750fa876cc1e54c7cb972db5e2f7271fc53d99/src/run.ts#L328`, therefore the usage of `branch`
+          title: 'chore(release): bump version and update changelog from `${{ steps.branch-check.outputs.branch_name }}`'
+          commit: 'chore(release): bump version and update changelog from `${{ steps.branch-check.outputs.branch_name }}`'
```

```suggestion
          title: 'chore(release): bump version and update changelog'
          commit: 'chore(release): bump version and update changelog'
```
the PR/commit goes on the release branch which can be seen in the review, so this seems redundant

---

The source and destination of the workflow and PR are the same so it's a bit confusing of a commit message. Maybe add the target in the PR description but not the title/commit message: `Creating a release on {branch name}`